modules/learn/pages/clusters-and-availability/xdcr-viewing-conflict-logs.adoc

```diff
@@ -0,0 +1,356 @@
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b
+WHERE META().id LIKE "crd%" and b.timestamp between "2025-06-04T18:15:16.363Z" AND "2025-06-09T07:26:40.326Z"
+----
+
+*Query example 4*: List all `docIds` and the number of times each `docId` had a logged conflict between 2 time periods.
+
+The following query may be used if you suspect that the same few documents are being updated multiple times concurrently in two different clusters, generating conflicts.
+
+[source,sqlpp]
+----
+SELECT b.docId AS DOCID, COUNT(1) AS num_times_seen
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b
+WHERE META(b).id LIKE "crd%" AND b.timestamp BETWEEN "2025-06-04T18:15:16.363Z" AND "2025-06-09T07:26:40.326Z"
+GROUP BY b.docId
+ORDER BY num_times_seen DESC LIMIT 20
+----
+
+The query result is as follows:
+
+----
+[
+  {
+    "DOCID": "4d9d18e7-5fa2-4eae-b32b-60d3d369da94",
+    "num_times_seen": 2   
+  },
+  {
+    "DOCID": "ae03ecde-203c-4a3b-a6d0-36b60f627c8a",
+    "num_times_seen": 1   
+  }
+]
+----
+
+*Query example 5*: In the following example, for each `docId`, you can see selected fields from the source and target documents.
+Also, the `docId` of the copies of the source and target documents in the conflict collection, if you want to retrieve the entire document.
+The only fields that you want to see from the documents that were in conflict are `country` and `note`.
+
+[source,sqlpp]
+----
+SELECT b.docId, b.srcDoc.id AS srcDocId, b.tgtDoc.id AS tgtDocId,
+(SELECT b2.country, b2.note
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b2 USE KEYS b.srcDoc.id)[0] AS src_doc_fields,
+(SELECT b3.country, b3.note
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b3 USE KEYS b.tgtDoc.id)[0] AS tgt_doc_fields
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b
+WHERE META(b).id LIKE "crd%" AND b.timestamp BETWEEN "2025-06-04T18:15:16.363Z" AND "2025-06-09T07:26:40.326Z"
+----
+
+The query result is as follows:
+----
+[
+   {
+    "docId": "ae03ecde-203c-4a3b-a6d0-36b60f627c8a",
+    "srcDocId": "src_1749454000005461807_e2f416ed317beb4f16a3587656b3d3450e1bec893cabebd3b9bb857d63824da4",
+    "tgtDocId": "tgt_1749454000005461807_e2f416ed317beb4f16a3587656b3d3450e1bec893cabebd3b9bb857d63824da4",
+    "src_doc_fields": {
+        "country": "UK",
+        “note”: “need review”
+      },
+    "tgt_doc_fields": {
+        "country": "CA",
+        “note”: “ok”
+      } 
+   }
+]
+----
+
+[#using-eventing-for-conflict-logs]
+== Using Eventing to Process Conflict Logs
+
+If you do not like the 3 documents format of the conflict record, then you can create a single document from the 3 documents (`crd_`, `src_`, `tgt_`) for easier processing by using Eventing.
+Here, it's assumed that the documents are small.
+If the documents are large such that the combined 3 documents are larger than the maximum xref:learn:clusters-and-availability/size-limitations.adoc[size limit].
+Then you can copy only a subset of the fields from the `src_` and `tgt_` documents to the custom single document.
```

Comment:
Line 162 isn't a complete sentence.

If the documents are large such that the combined 3 documents are larger than the maximum xref:learn:clusters-and-availability/size-limitations.adoc[size limit], you can copy a subset of the fields from the `src_` and `tgt_` documents to the custom single document.